python/sdk/prelude_sdk/controllers/scm_controller.py

```diff
@@ -0,0 +1,65 @@
+from prelude_sdk.controllers.http_controller import HttpController
+
+from prelude_sdk.models.codes import Control
+from prelude_sdk.models.account import verify_credentials
```

Comment:
im sure this is just copy pasted. but still 👀

Reply:
lol i can change it in both